src/devices/Ssd1306/Ssd1306.csproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <!--Disabling default items so samples source won't get build by the main library-->
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Command\ActivateScroll.cs" />
```

Comment:
I would also be fine with globing if you prefer to do that as well. By that I mean: `<Compile Include="Command\*.cs" />`. I'm fine with either approach so it's your choice :)

Reply:
Never heard of the term or have ever tried this before.  I'll update soon.